threema-connector-test/src_test/threema/connector/test/process/GetReceiverInfoTest.java

```diff
@@ -0,0 +1,132 @@
+package threema.connector.test.process;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
+import ch.ivyteam.ivy.bpm.engine.client.History;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmElement;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
+import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.environment.AppFixture;
+import ch.ivyteam.threema.mocks.ThreemaServiceMock;
+import threema.connector.receiverData;
+import util.LookupType;
+
+@IvyProcessTest(enableWebServer = true)
+public class GetReceiverInfoTest {
+	
+	private static final BpmProcess RECEIVER_INFO_PROCESS = BpmProcess.name("getReceiverInfo");
+	private static final String VALID_ID = "validId";
+	
+	@BeforeEach
```

Comment:
looks as if all your java-code was formatted with our bad defaults form the the Designer.
I'd import the preferences from the code ... and re-format all java code ... you can do it by right clicking on the src folder -> source -> format ... .after importing the prefs

Reply:
Applied formatting everywhere